book/src/configuration.md

```diff
@@ -201,18 +201,25 @@ Example:
 Enables automatic insertion of pairs to parentheses, brackets, etc. Can be a
 simple boolean value, or a specific mapping of pairs of single characters.
 
-To disable auto-pairs altogether, set `auto-pairs` to `false`:
+To disable auto-pairs altogether, set `config-type` to `false`:
 
 ```toml
-[editor]
-auto-pairs = false # defaults to `true`
+[editor.auto-pairs]
+config-type = false # defaults to `true`
```

Comment:
Not really nice to break a config without much of a need. Perhaps it's better to add `auto-pairs-overtype = <bool>` param?

Reply:
Do you mean adding an `auto-pairs-overtype = <bool>` param to the `Editor`? If so I can certainly do that. It seemed better to group it with the other `auto-pairs` configuration.

Reply:
Yeah I suggested this as an option but whether a breaking change like this would be accepted would be up to the maintainers. It feels better to group them together since we do that for e.g. soft-wrap and the file picker. It would be possible to avoid a breaking change by using an enum, but yeah it's up to the maintainers.

Reply:
The `config-type` naming is not very understandable to me, personally. Also I think having both `[editor.auto-pairs]` and `[editor.auto-pairs.<whatever_we_name_it>]` complicates configuration. It would be nice if overtype was co-located with the pair definitions.